src/modules/navigator/navigator_main.cpp

```diff
@@ -385,6 +385,7 @@ Navigator::task_main()
 				rep->current.loiter_radius = get_loiter_radius();
 				rep->current.loiter_direction = 1;
 				rep->current.type = position_setpoint_s::SETPOINT_TYPE_LOITER;
+				rep->current.cruising_speed = get_cruising_speed();
```

Comment:
If you're not going to use the default can you add throttle here as well? It matters for where you can command either or both.